preview_generator/preview/builder_factory.py

```diff
@@ -67,9 +81,6 @@ def get_file_mimetype(self, file_path: str, file_ext: str='') -> str:
                 .replace(': ', '')
```

Comment:
Use the following command line option : `mimetype --output-format %m <myfilepath>` instead of using `.replace()`